pkg/mdm/constants.go

```diff
@@ -0,0 +1,5 @@
+package mdm
+
+const (
+	BYODIdpCookieName = "__Host-FLEETBYODIDP"
```

Comment:
This was introduced in Magnus' PR, I copied it here as I need to reference that cookie name too.